consensus/reactor.go

```diff
@@ -338,7 +338,15 @@ func (conR *Reactor) ReceiveEnvelope(e p2p.Envelope) {
 		case *BlockPartMessage:
 			ps.SetHasProposalBlockPart(msg.Height, msg.Round, int(msg.Part.Index))
 			conR.Metrics.BlockParts.With("peer_id", string(e.Src.ID())).Add(1)
-			schema.WriteBlockPart(conR.traceClient, msg.Height, msg.Round, e.Src.ID(), msg.Part.Index, schema.TransferTypeDownload)
+			part, err := msg.Part.ToProto() // this conversion is only needed to get the part size
+			// consistent with how it is measured in the gossipDataRoutine
```

Comment:
[uber nit] since this comment is split across multiple lines, proposal to do something like this so that it's clear both lines apply to the code line on 341:

```go
// The .ToProto conversion is only needed to get the part size.
// This is consistent with how it is measured in the gossipDataRoutine.
part, err := msg.Part.ToProto()
```

Reply:
Incorporated you suggestion https://github.com/celestiaorg/celestia-core/pull/1067/commits/a82c5749254845b0e650304af0c59f31c12a4422